server/src/game-implementations/games/ddr.ts

```diff
@@ -141,6 +141,25 @@ export const DDR_SCORE_VALIDATORS: Array<ScoreValidator<"ddr:DP" | "ddr:SP">> =
 			default:
 		}
 	},
+	(s) => {
+		const { MARVELOUS, PERFECT, GREAT, OK } = s.scoreData.judgements;
```

Comment:
I probably should've caught this out when it was first written, but I don't think all of these key names should be capitalized. Maybe I can fix this in the future with a migration.